.github/workflows/images.yaml

```diff
@@ -113,14 +113,28 @@ jobs:
         id: build-time
         run: echo "now=$(date -u +'%Y-%m-%dT%H:%M:%SZ')" >> $GITHUB_OUTPUT
 
+      - name: Define proper repo
+        id: repo
+        run: |
+          case "${{ github.ref }}" in
+            refs/tags/v*)
+              IMAGE_NAME=ghcr.io/${{ github.repository }}
+              ;;
+            *)
+              IMAGE_NAME=ghcr.io/${{ github.repository }}-dev-builds
```

Comment:
I think `ghcr.io/${{ github.repository }}-dev` is clear enough, but it's just names.

Reply:
that's was my thought initially but it flet wrong so I added -builds, ca definitely rollback to -dev